File: C/1078.c
// Problema: 1078 - Tabuada | Resposta: Accepted
// Linguagem: C99 [+0s]     | Tempo: 0.000s

#include <stdio.h>

int main() {
    int multiplicador;
    scanf("%d", &multiplicador);

    for(int i = 1; i <= 10; i++){
        printf("%d x %d = %d\n", i, multiplicador, i*multiplicador);
    }

    return 0;
}

File: JavaScript/1078.js
// Problema: 1078 - Tabuada          | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s] | Tempo: 0.027s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
const multiplicador = parseInt(input.trim());
for(let i = 1; i <= 10; i++){
    console.log(`${i} x ${multiplicador} = ${i * multiplicador}`);
}